src/lib/pickles/plonk_checks/plonk_checks.mli

```diff
@@ -82,7 +78,7 @@ val scalars_env :
   -> endo:'t
   -> mds:'t array array
   -> field_of_hex:(string -> 't)
-  -> domain:< generator : 't ; vanishing_polynomial : 't -> 't ; .. >
+  -> domain:'t domain
```

Comment:
Why does this work, given that  `scalar_env` in `.ml` stays the same?

Reply:
yes, `generator` attribute is required in the function, see `domain#generator`.

Reply:
It doesn't: the CI is clear. I was too fast, mistook `size` for `generator`, and brainfarted. This change is thus reverted.